### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by samif1**
Added a .gitignore file.
Set up a linter for HTML and CSS.
Added descriptive README file 
Added .html file that includes a header with the text "Hello Microverse!". 
Added .css file that includes one class with my styles for header.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (mckent05) [inline README.md:25]: - In order to make your readme look more professional, please include steps on how to clone your project locally 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @samif1,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- Good job setting up your first project well 👍 👍 
- Your PR title and PR summary are very descriptive 👏 👏 
- All linter checks are passing ✔️ ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by samif1** [comment]
You mean i have to attach a file how the linter setup is installed?

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada:  :heavy_check_mark: 

Hi @samif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Fait README file. :heavy_check_mark:




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by samif1** [comment]
Thanks a lot.

**Pull request merged**